client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -531,6 +532,20 @@ export const confirmPackages = ( orderId, siteId ) => ( dispatch, getState ) =>
 	tryGetLabelRates( orderId, siteId, dispatch, getState );
 };
 
+export const saveCustoms = ( orderId, siteId ) => {
+	return {
+		type: WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_CUSTOMS,
+		siteId,
+		orderId,
+	};
+};
+
+export const confirmCustoms = ( orderId, siteId ) => ( dispatch, getState ) => {
```

Comment:
The action `confirmCustoms` seems to be created to be a click handler for the customs submit step button. Can we move it to the component file instead?

Reply:
That's not what we've been doing so far. Even if an action maps directly to a user interaction, we normally put it with the others in the `actions.js` file. I think it helps keeping UI and logic a bit more separated. Plus, `tryGetLabelRates` isn't `export`ed.

Reply:
Ok. I don't fully agree with that, but let's leave as is for now, since our part of the codebase follows this pattern.

About the UI/logic separation of concerns - as an example when a UI change is needed (e.g. we don't want to toggle the step anymore), or if we need to confirm customs in some other component , we'll need to touch 'logic' code in `actions.js`, so putting the click handler there makes the two components coupled.